Create UI Structure

hannessolo opened the pull request.

icaraps commented on pages/_app.tsx:
```diff
@@ -1,8 +1,15 @@
+    return (
+        // SSR Provider is required for SSR/Static rendering
+        <SSRProvider>
+            <Provider theme={defaultTheme}>
```
Can you pls use the light theme ? As the parent shell is also using the light theme by default

icaraps commented on .gitignore:
```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
+
+.idea
```
This can be added in your global gitignore so it's not required here

icaraps commented on Slides/SummarySlide.tsx:
```diff
@@ -0,0 +1,20 @@
+  const {next, prev, formData} = useContext(FormContext)
+
+  return <SlideLayout>
+    <h1>Ok, thanks User</h1>
```
Can you use spectrum's typography instead here ? You'll have to import the css package https://opensource.adobe.com/spectrum-css/typography.html 
We'll also add the adobe font later.

hannessolo replied:
I'm not sure how to do this, I don't think I can import that library into nextjs?

icaraps commented on Slides/SummarySlide.tsx:
```diff
@@ -0,0 +1,20 @@
+
+  return <SlideLayout>
+    <h1>Ok, thanks User</h1>
+    <span>Here’s what we will record for your trial:</span>
```
Can you use a layout component here to display the data ?

icaraps commented on components/Slides.module.css:
```diff
@@ -0,0 +1,31 @@
+    bottom: 0;
+}
+
+.actionButtonContainer > * {
```
Maybe you can be more specific in your selector here

icaraps reviewed (changes requested):
LGTM ! There's one thing major I'd like to change if possible. Could you think of a way to extract the raw data (questions, responses, text) into another file (eg json) to separate the layout from the actual data ? 
I also noticed a few spacing issues eg on slide 2 between the select and the buttons. Or alignment issues, eg radio boxes are not vertically aligned. 
In general, I think it would make sense to increase the container width and add more spacing while making sure it's responsive.

hannessolo commented:
- I improved the layout somewhat, but will wait before doing more in case Ron decides to change it, and focus on functionality first
- I'm not sure how to import the css typography library you commented
- I extracted the long strings, but won't do smaller ones yet, because I have a branch where I changed some of these - so I don't have to do it all twice
- I will move the repo to git corp once we merge this
